examples/ignition/k8s-master.yaml

```diff
@@ -0,0 +1,551 @@
+                                    }
+                                },
+                                {
+                                    "args": [
+                                        "-cmd=nslookup kubernetes.default.svc.cluster.local localhost >/dev/null",
+                                        "-port=8080"
+                                    ],
+                                    "image": "gcr.io/google_containers/exechealthz:1.0",
+                                    "name": "healthz",
+                                    "ports": [
+                                        {
+                                            "containerPort": 8080,
+                                            "protocol": "TCP"
+                                        }
+                                    ],
+                                    "resources": {
+                                        "limits": {
+                                            "cpu": "10m",
+                                            "memory": "20Mi"
+                                        }
+                                    }
+                                }
+                            ],
+                            "dnsPolicy": "Default",
+                            "volumes": [
+                                {
+                                    "emptyDir": {},
+                                    "name": "etcd-storage"
+                                }
+                            ]
+                        }
+                    }
+                }
+            }
+        - path: /srv/kubernetes/manifests/kube-dns-svc.json
+          contents: |
+            {
+              "apiVersion": "v1",
+              "kind": "Service",
+              "metadata": {
+                "name": "kube-dns",
+                "namespace": "kube-system",
+                "labels": {
+                  "k8s-app": "kube-dns",
+                  "kubernetes.io/name": "KubeDNS",
+                  "kubernetes.io/cluster-service": "true"
+                }
+              },
+              "spec": {
+                "clusterIP": "{{.k8s_dns_service_ip}}",
+                "ports": [
+                  {
+                    "protocol": "UDP",
+                    "name": "dns",
+                    "port": 53
+                  },
+                  {
+                    "protocol": "TCP",
+                    "name": "dns-tcp",
+                    "port": 53
+                  }
+                ],
+                "selector": {
+                  "k8s-app": "kube-dns"
+                }
+              }
+            }
+        - path: /srv/kubernetes/manifests/kube-system.json
+          contents: |
+            {
+              "apiVersion": "v1",
+              "kind": "Namespace",
+              "metadata": {
+                "name": "kube-system"
+              }
+            }
+        - path: /opt/init-flannel
+          mode: 320
```

Comment:
"mode must be properly specified as a decimal value" Are the docs incorrect? I converted to decimal. https://coreos.com/ignition/docs/latest/configuration.html